models/bamf_ct_liver_tumor/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM mhubai/base:latest
+
+# FIXME: set this environment variable as a shortcut to avoid nnunet crashing the build
+# by pulling sklearn instead of scikit-learn
+# N.B. this is a known issue:
+# https://github.com/MIC-DKFZ/nnUNet/issues/1281 
+# https://github.com/MIC-DKFZ/nnUNet/pull/1209
+ENV SKLEARN_ALLOW_DEPRECATED_SKLEARN_PACKAGE_INSTALL=True
+
+# Install nnunet and platipy
+RUN pip3 install --no-cache-dir nnunetv2==2.0 \
+        dcm2niix==1.0.20220715
```

Comment:
Why do we need to install a specific version that differs from the one shipped in the base image? Please elaborate.

Reply:
> mhubai/base image uses dcm2niix==1.0.20181125 which causes error for NiftiConverter . One example for this issue can be found while converting `s3://idc-open-data/aafd2a22-6236-4eff-b541-1dfd8de923e8/*`

I found this issue while testing in #[86](https://github.com/MHubAI/models/pull/86#discussion_r1697405895)

Reply:
Changes to be made based on the same discussion in the pull request #86 comments [link](https://github.com/MHubAI/models/pull/86#discussion_r1697405895)

Reply:
I have removed installation of `dcm2niix==1.0.20220715` as per the comments in #86